app/controllers/greetings_controller.rb

```diff
@@ -0,0 +1,52 @@
+class GreetingsController < ApplicationController
+  before_action :set_greeting, only: %i[show update destroy]
+
+  # GET /greetings
+  def index
+    @greetings = Greeting.all
+
+    render json: @greetings
+  end
```

Comment:
- [ ] 👍 Great job on incorporating the random greeting feature on the front end! 😊 However, please keep in mind that the API endpoint is expected to provide a random greeting. You can achieve this by defining the index action in your GreetingsController to return a random greeting, just like this:
```
class GreetingsController < ApplicationController
  def index
    @random_greeting = Greeting.order("RANDOM()").first
    render json: { greeting: @random_greeting&.content }
  end
end

```